main.py

```
@@ -1,39 +1,81 @@
+            try:
+                with open(LOCK_PATH, "rb") as lock_file:
+                    raw_data = lock_file.read()
+                    if not raw_data:
+                        Constants.log.warning("[daemon] 锁文件为空")
+                        raise psutil.NoSuchProcess(0)
+
+                    try:
+                        pid = int(base64.b85decode(raw_data).decode())
+                    except Exception as e:
+                        Constants.log.error(f"[daemon] 锁文件解析失败: {str(e)}")
+                        raise psutil.NoSuchProcess(pid)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)